mal/useranimelist.go

```diff
@@ -83,6 +83,8 @@ type AnimeListStatus struct {
 	RewatchValue       int         `json:"rewatch_value"`
 	Tags               []string    `json:"tags"`
 	Comments           string      `json:"comments"`
+	StartDate          string      `json:"start_date,omitempty"`
+	FinishDate         string      `json:"finish_date,omitempty"`
```

Comment:
This struct is only used to unmarshal from the MAL API so there's no need for omitempty here.
```suggestion
	StartDate          string      `json:"start_date"`
	FinishDate         string      `json:"finish_date"`
```